packages/charts/react-charting/src/components/HeatMapChart/HeatMapChart.types.ts

```diff
@@ -108,6 +108,11 @@ export interface IHeatMapChartProps extends Pick<ICartesianChartProps, Exclude<k
    * The prop used to define the culture to localized the numbers
    */
   culture?: string;
+
+  /**
+   *@default false
+   *Used for showing complete y axis lables   */
+  showYAxisLables?: boolean;
```

Comment:
>showYAxisLables [](http://example.com/codeflow?start=2&length=15)

his prop is already part of cartesian. Should not be needed here.

Reply:
yes.